frontend/src/components/AppRoutes.jsx

```diff
@@ -3,6 +3,7 @@ import React from "react"
 import { Route, Routes } from "react-router-dom"
 import NewBookForm from "../features/books/NewBookForm"
 import BookDetails from "../features/books/BookDetails"
+import EditBookForm from "../features/books/EditBookForm"
 
 
 function AppRoutes(){
```

Comment:
@nshelokar-vf, is there any specific reason to use functional component & not arrow function for component?

Reply:
Not specifically any reason but from next time I will try to use Arrow functions for component.